### PR TITLE
Add dts to FilePicker component

### PR DESCRIPTION
**Pull request by OndrejCholeva**
usage in tests:
``` coffeescript
filePickerElement.sendKeys('absolute/path') #simulates user selecting file by filepicker
```

### CODE REVIEW

**Comment by NamrataTrivedi** [comment]
LGTM :open_file_folder:

**Comment by omgaz** [comment]
LGTM

File: src/components/adslotUi/FilePickerComponent.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ class FilePickerComponent extends React.Component {
                 type="file"
                 onChange={this.onChange}
                 accept={this.props.filter}
+                data-test-selector={this.props.dts}
```
Comment (vinteo) [inline src/components/adslotUi/FilePickerComponent.js]: should we put this on the main `div` and classNames/dts on individual elements so they can be targeted as well or is that overkill?

Comment (OndrejCholeva) [inline src/components/adslotUi/FilePickerComponent.js]: I was thinking about that, but for e2e purpose we only need to target the `<input type="file">` element (protractor sendKeys simulates both selecting and submitting action).

**Comment by OndrejCholeva** [comment]
@omgaz / @NamrataTrivedi / @vinteo  merge?

**Pull request merged**